packages/components/package.json

```diff
@@ -71,7 +76,7 @@
     "sass-loader": "^8.0.2",
     "ts-jest": "^24.1.0",
```

Comment:
We may want to use `babel-jest` for tests, so that the same compilation happens in code as happens in builds.